CloudFormation/Docker/windows-dotnet-web-application/windows-dotnet-web-application-pipeline.yml

```diff
@@ -0,0 +1,302 @@
+      # These variables will be replaced with Image Builder generated scripts at build time.
+      # In this Dockerfile, `DOTNET_SYSTEM_GLOBALIZATION_INVARIANT` is required to run .NET, and the entrypoint
+      # is configured with the .NET application.
+      DockerfileTemplateData: !Sub |
+        FROM {{{ imagebuilder:parentImage }}}
+        {{{ imagebuilder:environments }}}
+        {{{ imagebuilder:components }}}
+        EXPOSE ${ApplicationPort}
+        ENTRYPOINT ["dotnet", "c:\\windows-webapp\\${WebsiteName}.dll"]
+      WorkingDirectory: C:/
+
+  # A CloudWatch LogGroup that maps to where the image creation logs will be published.
+  # This ensures the retention is configured, and that the group is removed on stack deletion.
+  RecipeLogGroup:
+    Type: AWS::Logs::LogGroup
+    Properties:
+      LogGroupName: !Sub '/aws/imagebuilder/${ResourceName}'
+      RetentionInDays: 7
+
+  # Allows you to specify the name and description of your output container image and settings for tagging and sharing
+  # to a target ECR repository in a specific region.
+  # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-distributionconfiguration.html
+  Distribution:
+    Type: AWS::ImageBuilder::DistributionConfiguration
+    Properties:
+      Name: !Ref ResourceName
+      Description: 'This distribution configuration will deploy our container image to the desired target ECR repository in the current region'
+      Distributions:
+        - Region: !Ref 'AWS::Region'
+          ContainerDistributionConfiguration:
+            TargetRepository:
+              Service: ECR
+              RepositoryName: !Ref TargetECRRepository
+            # By default, if no container tags are specified, the "Image tags" value in ECR will reflect the image build version (e.g. 1.0.0-1),
+            # which can be found by navigating to EC2 ImageBuilder in the AWS Management Console.
+            ContainerTags:
+              - !Ref ImageTag
+
+  # The pipeline is scheduled to run a build at 10:00AM Coordinated Universal Time (UTC) every day.
+  # The build will only run if dependencies have been updated.
+  # https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-imagebuilder-imagepipeline.html
+  Pipeline:
+    Type: AWS::ImageBuilder::ImagePipeline
+    Properties:
+      ContainerRecipeArn: !Ref Recipe
+      Description: A pipeline to automate creation of the container
+      DistributionConfigurationArn: !Ref Distribution
+      ImageTestsConfiguration:
+        ImageTestsEnabled: false
+        TimeoutMinutes: 60
+      InfrastructureConfigurationArn: !Ref Infrastructure
+      Name: !Ref ResourceName
+      Schedule:
+        PipelineExecutionStartCondition: EXPRESSION_MATCH_AND_DEPENDENCY_UPDATES_AVAILABLE
+        ScheduleExpression: 'cron(0 10 * * ? *)'
+      Status: ENABLED
+      Tags:
+        Purpose: PowerShellSummitContainerDemo
```

Comment:
We should update the tag for this sample.

Reply:
Tag has been dropped.